.prettierignore

```diff
@@ -0,0 +1,4 @@
+*.js
```

Comment:
/nit Can these be alphabetised / in the order they'd appear in the IDE? I mostly just want `coverage` before `dist`

```
coverage
dist
CHANGELOG.md
*.js
```

Reply:
Yep!